### PR TITLE
Do not automatically pull binaries during the "branch" command

### PR DESCRIPTION
**Pull request by sergii-s**
#168

### CODE REVIEW

**Comment by pchalamet** [comment]
:white_check_mark: [Build full-build 4.2.24 completed](https://ci.appveyor.com/project/full-build/full-build-shu78/build/4.2.24) (commit https://github.com/full-build/full-build/commit/a9582aab87 by @sergii-s)

**Comment by pchalamet** [comment]
PR #186 must be tested first - this is related because when you branch, you also want to get matching source code (and get broken .anthology). This shall solve this problem.

I also suspect there is maybe another bug related to the way bin/.anthology is used. It shall never be used as is but as the foundation on which all other available repo/.anthology are applied on. It's shall be the case but something might be broken there.

**Comment by sergii-s** [comment]
If I want to build ALL from zero I dont want nor old binairies nor old anthology.

**Comment by pchalamet** [comment]
Agree. But until you clone all sources you have *no* way to know you will end up building everything or not.

**Comment by pchalamet** [comment]
if the idea is to build from scratch - implementing clean binaries instead would be more helpful ?

**Comment by sergii-s** [comment]
> Agree. But until you clone all sources you have no way to know you will end up building everything or not.

Yes, but still no need of the old build result. Just need to pull + branch master repo and then all others.

> implementing clean binaries instead would be more helpful ?

I don't think so. There are another related problems, for example build sources when last build is corrupted (result missing but master repo is tagged). The only way to do with actual solution is to mock last build result or another manual action, but the main goal is build all from sources

File: src/FullBuild/CLI/CommandLine.fs [hunk 1]
```diff
@@ -548,7 +549,7 @@ let Parse (args : string list) : Command =
     | Token Token.Build :: cmdArgs -> cmdArgs |> commandBuild "Release" false false None
     | Token Token.Rebuild :: cmdArgs -> cmdArgs |> commandBuild "Release" true false None
     | Token Token.Checkout :: cmdArgs -> cmdArgs |> commandCheckout
-    | Token Token.Branch :: cmdArgs -> cmdArgs |> commandBranch
+    | Token Token.Branch :: cmdArgs -> cmdArgs |> commandBranch false
```
Comment (pchalamet) [inline src/FullBuild/CLI/CommandLine.fs]: Would prefer true to try first.

**Review by pchalamet — CHANGES_REQUESTED** [review]

**Comment by pchalamet** [comment]
:white_check_mark: [Build full-build 4.2.26 completed](https://ci.appveyor.com/project/full-build/full-build-shu78/build/4.2.26) (commit https://github.com/full-build/full-build/commit/5503655be3 by @sergii-s)

**Review by pchalamet — APPROVED** [review]

**Pull request merged**